enterprise/utils.py

```diff
@@ -1952,14 +1967,30 @@ def enroll_subsidy_users_in_courses(enterprise_customer, subsidy_users_info, dis
         'failures': [],
     }
     for subsidy_user_info in subsidy_users_info:
-        user_email = subsidy_user_info.get('email').strip().lower()
+        user_id = subsidy_user_info.get('user_id')
+        user_email = subsidy_user_info.get('email').strip().lower() if 'email' in subsidy_user_info else None
```

Comment:
Does this need to be None? Alternatively you could do `user_email = subsidy_user_info.get('email', '').strip().lower()`

Reply:
I would prefer to keep this None for consistency with user_id, but what I was able to do was slightly shorten the line by using `subsidy_user_info['email'].strip().lower()` which is still protected by the following condition.